src/stylesheets/components/_quit-survey-modal.scss

```diff
@@ -0,0 +1,78 @@
+.quit-survey-modal {
+  &__container {
+    position: fixed;
+    top: 0;
+
+    width: 100%;
+    height: 100%;
+
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  &__content {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+
+    display: flex;
+    flex-flow: column;
+
+    width: rem(467px);
+    height: rem(178px);
+    padding: rem(24px);
+
+    background: map_get($brand-black, 'primary');
+    border-radius: rem(14px);
+    transform: translate(-50%, -50%);
+  }
+
+
+  &__header {
+    display: flex;
+    margin-bottom: rem(8px);
+
+    color: map_get($brand-white, 'primary');
+    font-weight: map_get($font-weight, 'bolder');
+    font-size: map_get($font-sizes, 'xl');
+    line-height: rem(28px);
+  }
+
+  &__body {
+    display: flex;
+
+    color: map_get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'md');
+    line-height: rem(22px);
+
+    opacity: 0.7;
+  }
+
+  &__footer {
+    display: flex;
+    flex: 1 1 auto;
+    align-items: flex-end;
+
+    width: rem(188px);
+    margin-left: auto;
+  }
+
+  &__yes-button {
+    @include warning-actions();
+
+    margin-right: rem(4px);
```

Comment:
@rafayet-monon Fromt eh comment above we can also remove the `margin-right` and `margin-left` from both buttons and add only the `margin-left` to the `.modal.button:not(:first-child)` to apply the style to every button but not the first one
```scss
.modal {
  .button:not(:first-child) {
    margin-left: rem(8px);
  }
}
```

Reply:
@carryall Fixed reviews in [caa40a6](https://github.com/rafayet-monon/surveyor/pull/30/commits/caa40a6e7d20bc7d105634f5d249b3feadeb5f63)